### PR TITLE
Help provided on the file format specified

### PR DESCRIPTION
**Pull request by federicomarini**
... as in #53 

We'd need an extra screenshot if we want to show also how it looks.
Do we?

### CODE REVIEW

File: inst/extdata/howto_datainput.md
Comment (jharenza) [inline inst/extdata/howto_datainput.md]:
```suggestion
- The file should contain a **header row**, with minimal column names: `Sample`, `LeftBreakpoint`, `RightBreakpoint`, `FusionName`, `Gene1A`, and `Gene1B`, with rows formatted as shown in the example below.
- If your data in stored in Excel, export the file as tab-separated format for compatibility with `shinyFuse`
```

File: inst/extdata/howto_datainput.md
Comment (jharenza) [inline inst/extdata/howto_datainput.md]:
```suggestion
Below is a screenshot of for a minimal file which will be correctly loaded by `shinyFuse`.
To take full advantage of all features, we recommend running fusion analysis, annotation, followed by `annoFuse` as noted [here](https://github.com/d3b-center/annoFuse#install-package).
For a fully annotated input file example, see the [`PutativeDriverAnnoFuse.tsv`](https://github.com/d3b-center/annoFuse/blob/master/inst/extdata/PutativeDriverAnnoFuse.tsv) provided bundled with the package.
```

Note: that file link will be broken until we merge with master, unless @kgaonkar6, you want to push the file to master right now (I am ok with that), while keeping putative oncogenic until exemplary is merged.

Comment (jharenza) [reply inst/extdata/howto_datainput.md]: Please add one of these images as the example (the users who will need this on the shiny server will likely be non-computational and I think the current image is hard to discern for that audience). Also, this image highlights some oncogenic fusions.

Screen Shot 2020-09-04 at 11 50 20 AM

Screen Shot 2020-09-04 at 11 53 40 AM

Screen Shot 2020-09-04 at 11 52 46 AM

**Review by jharenza — CHANGES_REQUESTED** [review]

File: inst/extdata/howto_datainput.md
Comment (federicomarini) [reply inst/extdata/howto_datainput.md]: I like this examples with minimal sets of columns. 
And of the three, I like most the solarized one (number 3) - happy to replace the current one with this!

File: inst/extdata/howto_datainput.md
Comment (jharenza) [inline inst/extdata/howto_datainput.md]:
```suggestion
Below is a screenshot of a minimal file which will be correctly loaded by `shinyFuse`.
```
typo, my bad

File: inst/extdata/howto_datainput.md
Comment (jharenza) [inline inst/extdata/howto_datainput.md]:
```suggestion
- `shinyFuse` accepts **tab-separated text files**, as commonly produced by the workflow of `annoFuse`.
```

File: inst/extdata/howto_datainput.md
Comment (jharenza) [inline inst/extdata/howto_datainput.md]:
```suggestion
- If your data are stored in Excel, export the file as tab-separated format for compatibility with `shinyFuse`.
```
typo, probably also my fault

**Review by jharenza — APPROVED** [review]
approve once changes are made

File: inst/extdata/howto_datainput.md
Comment (jharenza) [inline inst/extdata/howto_datainput.md]:
```suggestion
- The file should contain a **header row**, with minimal column names: `Sample`, `LeftBreakpoint`, `RightBreakpoint`, `FusionName`, `Gene1A` (5' gene), and `Gene1B` (3' gene), with rows formatted as shown in the example below.
```

**Review by jharenza — CHANGES_REQUESTED** [review]

**Comment by federicomarini** [comment]
Good to go!

**Pull request merged**
